### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by fstanis**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -23,16 +23,16 @@ buildscript {
         versionCode = 10002
 
         // SDK and tools
-        compileSdkVersion = 28
+        compileSdkVersion = 29
```
Comment (pflammertsma) [inline build.gradle]: I think we should stick with the recommendations from https://developer.android.com/training/wearables/apps/creating#wear-project and keep this to 28.
```suggestion
        compileSdkVersion = 28
```

File: build.gradle [hunk 2]
```diff
@@ -23,16 +23,16 @@ buildscript {
...
         minSdkVersion = 23
-        targetSdkVersion = 28
+        targetSdkVersion = 29
```
Comment (pflammertsma) [inline build.gradle]: Here as well 28.
```suggestion
        targetSdkVersion = 28
```

**Review by pflammertsma — COMMENTED** [review]
For Wear OS, we may want to stick with targetSdkVersion 28.

**Comment by fstanis** [comment]
Fixed.

**Pull request merged**